src/common/intrusive_lru.h

```diff
@@ -170,6 +172,22 @@ class intrusive_lru {
     }
   }
 
+  /*
+   * Clears all elements
+   */
+  void clear() {
```

Comment:
The elements that were cleared but not deleted yet are still accessible via get()/get_or_create(), right?

Reply:
Um, not really, they are not accessible through get()/get_or_create().